build.gradle

```diff
@@ -26,8 +26,8 @@ buildscript {
     classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.4'
     classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:4.0.0'
     classpath 'com.android.tools.build:gradle:1.5.0'
-    classpath 'com.redhat.ceylon.gradle:android:0.0.1-SNAPSHOT'
-//    classpath "com.athaydes.gradle.ceylon:ceylon-gradle-plugin:1.1"
+//    classpath 'com.redhat.ceylon.gradle:android:0.0.1-SNAPSHOT'
```

Comment:
@FroMage from the information in `mapping.txt`, I cannot find easily what needs to be compiled... I would need to know the imports of each modified file to know that. To parse the imports of the ceylon files is not trivial, I think... or is it? Maybe just parsing the import lines up to the braces should be enough to know at least which packages a file depends on?

If you think this info can be automatically generated, then I can build a tree of files that need compilation... then I guess I can call `ceylon compile` with a list of files to compile... that should work.

Let me know if I should write a basic import parser, or you want to generate the dependency information more reliably somehow...